src/bots/modchat/reactionHandler.js

```diff
@@ -28,8 +28,8 @@ let alertModChannel = async (reaction, user, action) => {
   let modChannel = await getModChannel();
 
   let url = reaction.message.url;
-  let title = `${reaction.message.author.username}#${reaction.message.author.discriminator}`;
-  let userName = `${user.username}#${user.discriminator}`;
+  const { author } = reaction.message;
+  let userName = `${author.username}#${author.discriminator}`;
```

Comment:
This could be my own ignorance, but how comes `author = reaction.message` if there exists `reaction.message.author` ?

Maybe there is a slight misnaming of the variable?

Reply:
It's using [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#object_destructuring) syntax to avoid code repetition.

For example, I could also write this as:
`const author = reaction.message.author;`
but we'd be repeating `author` twice, so we can simply do
`const { author } = reaction.message;`
which requires less repetition.

I could also rename the variable if I wanted to while destructuring:
`const { user: author } = reaction.message;`
and now `user` will be same as `reaction.message.author`

Reply:
Neat, I learned something today. Thanks for the crash course! <3